File: pythonclass/pythonclasswork/BankApp.java
import java.util.Scanner;
public class BankApp {
	public static void main(String[] args){
	Scanner input = new Scanner(System.in);

	int count = 0;
	int userInput = 0;
	int totalDeposit = 0;
	int balance =0;
	int withdraw = 0;
	
	while (userInput != -1){
	System.out.println("1. Deposit \n 2. Withdraw \n 3. Balance \n -1. Exit");

	System.out.println("Enter a number: ");
	userInput = input.nextInt();

	if (userInput == 1){
		System.out.println("Enter DEPOSIT amount: ");
		int deposit = input.nextInt();

		if (deposit < 0)
		System.out.println("Invalid amount");
		totalDeposit += deposit;}
	
	else if (userInput == 2){
		System.out.println("Enter WITHDRAWAL amount: ");
		withdraw = input.nextInt();
		if (withdraw > totalDeposit)
			System.out.println("Invalid amount input");
		else if (withdraw < 0)
			System.out.println("Invalid amount input");
		
		else
			System.out.println("Withdrawal successfull");

		}
	
	else if (userInput == 3){
       	balance = totalDeposit - withdraw;
        System.out.println(balance);}

	else
		break;
		}
	
		
	
	}
}

	

File: pythonclass/pythonclasswork/bankapp.py
count = 0
user_input = 0
total_deposit = 0

while user_input != -1:
    print('1.Deposit', '\n', '2. Withdraw', '\n', '3.Balance')

    user_input = int(input('Enter a number: '))

    if user_input ==1:
        deposit = int(input('Enter a deposit amount: '))
        total_deposit += deposit
                
    elif user_input == 2:
        withdraw = int(input('Enter a withdrawal amount: '))
        if withdraw > total_deposit:
            print('invalid amoount input')
        else:
            print('Withdrawal successful')
    elif user_input == 3:
        balance = total_deposit - withdraw
        print(balance)

    else:
        break

            

    
